incubator/hnc/e2e/tests/delete_crd_test.go

```diff
@@ -62,4 +63,21 @@ var _ = Describe("Delete-anchor-crd", func() {
 		// verify
 		mustRun("kubectl get -oyaml rolebinding foo -n", nsChild)
 	})
+
+	It("should delete the HC CRD, anchor CRD, and verify that the HNC CRDs are gone", func() {
+		// set up
+		mustRun("kubectl create ns", nsParent)
+		mustRun("kubectl hns create", nsChild, "-n", nsParent)
+		// test
+		mustRun("kubectl delete crd hierarchyconfigurations.hnc.x-k8s.io &")
+		time.Sleep(1 * time.Second)
+		mustRun("kubectl delete crd subnamespaceanchors.hnc.x-k8s.io &")
+		mustRun("kubectl delete crd hncconfigurations.hnc.x-k8s.io &")
+		// Sleeping for 10s to give HNC the chance to fully delete everything (5s wasn't enough).
+		time.Sleep(10 * time.Second)
+		// Verify that the HNC CRDs are gone (if nothing's printed, then they are).
+		stdout, err := runCommand("kubectl get crd | grep hnc")
+		Expect(err).Should(BeNil())
+		Expect(stdout).Should(BeNil())
```

Comment:
Here's a good chance to use `Eventually`:

```
Eventually(func() error{
  stdout, err := runCommand("kubectl get crd | grep hnc")
  if err != nil {
    return err
  }
  if stdout != nil {
    return error.New(string(stdout))
  }
}).Should(Succeed())
```

lmk if you need any clarity on how that works.

Reply:
Done